packages/components/src/EditableText/EditableText.test.js

```diff
@@ -90,7 +90,7 @@ describe('InlineForm', () => {
 		expect(
 			wrapper
 				.find('Action')
-				.first()
+				.at(1)
```

Comment:
This change seems curious

Reply:
First is index 0. To follow the guidelines, I inverted the two buttons.

Reply:
My brain was in a lua one based index, sorry --'